src/jsx/import/result/current-result.jsx

```diff
@@ -256,15 +258,15 @@ var CurrentResultDisplay = React.createClass({
                     defaultValue=''
                 />
                 <Submit cssClass='btn' />
-            </form>
+            </form>);
```

Comment:
I think the closing parentheses, should be on their own lines, unless you can show me eslint preference for otherwise.